Week2/database.js

```diff
@@ -0,0 +1,112 @@
+const mysql = require('mysql');
+const prompts = require('prompts');
+const { connection, execQuery } = require('./connection');
+const { question1, question2, question3, question4, question5 } = require('./queries');
+
+const getCapitalOfCountry = async () => {
+  try {
+    const response = await prompts({
+      type: 'text',
+      name: 'country',
+      message: `What is the capital of ?`,
+    });
+
+    const capitalQuery = await execQuery(question1, response.country);
+    const capitalOfCountry = capitalQuery[0].capital;
+    console.log(capitalOfCountry);
+  } catch (error) {
+    console.log(`There is no country for this input. ${error.message}`);
+  }
+};
+
+const getLanguagesInRegion = async () => {
+  try {
+    const response = await prompts({
+      type: 'text',
+      name: 'region',
+      message: `List all the languages spoken in the region ...?`,
+    });
+    const userInput = response.region.charAt(0).toUpperCase() + response.region.slice(1);

```

Comment:
You don't need this, sql is not case sensitive.

Reply:
When I write this code which is Js , Mysql did not accept lower case. So I implemented this code. But I have changed it. Now no need. Thank you